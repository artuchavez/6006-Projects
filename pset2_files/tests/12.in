gradebook-small
5
A,B,C,D,E,F,G,H,I,J
A,B,C,D,E,F,G,H,I,F
12,9,6,3,5,1,12,6,12,1
4.8,4.9,3.2,3.8,4.3,3.9,2.1,1.7,2.5,4.9

