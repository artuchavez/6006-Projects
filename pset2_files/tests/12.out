A,B,C,D,E,F,G,H,I,J
4.8,4.9,3.2,3.8,4.3,4.4,2.1,1.7,2.5,0
2.1,2.5,3.2,3.8,4.3